_rest_config.php

```diff
@@ -366,18 +411,30 @@ public static function skipApiAuth($resource): bool
 
     public static function apiLog($response = '', $requestBody = ''): void
     {
+        $logResponse = $response;
+
         // only log when using standard api calls (skip when using local api calls from within OpenEMR)
         //  and when api log option is set
         if (!$GLOBALS['is_local_api'] && !self::$notRestCall && $GLOBALS['api_log_option']) {
             if ($GLOBALS['api_log_option'] == 1) {
                 // Do not log the response and requestBody
-                $response = '';
+                $logResponse = '';
                 $requestBody = '';
             }
+            if ($response instanceof ResponseInterface) {
+                if (self::shouldLogResponse($response)) {
+                    $body = $response->getBody();
+                    $logResponse = $body->getContents();
+                    $body->rewind();
+                } else {
+                    $logResponse = '';
```

Comment:
In this case would log a message that not appropriate to log since a binary (then it was logged that something was sent)

Reply:
Not sure I understand.  Make $logResponse = 'logging skipped as content is not JSON'; or something like that?

Reply:
yep, then when audit will know that something went out